config/routes.rb

```diff
@@ -56,10 +56,17 @@
   #     resources :products
   #   end
 
+  resources :friendships
+
+  get 'friendship/my_friends' => 'friendships#my_friends', as: :my_friends
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.